docs/sources/api/_index.md

```diff
@@ -1180,18 +998,341 @@ For more information, please check out the Prometheus [alerts](https://prometheu
 
 ## Compactor
 
-### `GET /compactor/ring`
+### Compactor ring status
+
+```
+GET /compactor/ring
+```
 
 Displays a web page with the compactor hash ring status, including the state, health, and last heartbeat time of each compactor.
 
-### `POST /loki/api/v1/delete`
+### Request log deletion
+
+```
+POST /loki/api/v1/delete
+PUT /loki/api/v1/delete
+```
 
 Create a new delete request for the authenticated tenant. More details can be found in the [logs deletion documentation](../operations/storage/logs-deletion.md#request-log-entry-deletion).
 
-### `GET /loki/api/v1/delete`
+Log entry deletion is supported _only_ when the BoltDB Shipper is configured for the index store.
+
+Query parameters:
+
+* `query=<series_selector>`: query argument that identifies the streams from which to delete with optional line filters.
+* `start=<rfc3339 | unix_timestamp>`: A timestamp that identifies the start of the time window within which entries will be deleted. If not specified, defaults to 0, the Unix Epoch time.
+* `end=<rfc3339 | unix_timestamp>`: A timestamp that identifies the end of the time window within which entries will be deleted. If not specified, defaults to the current time.
+
+A 204 response indicates success.
+
+URL encode the `query` parameter. This sample form of a cURL command URL encodes `query={foo="bar"}`:
+
+```
+curl -g -X POST \
+  'http://127.0.0.1:3100/loki/api/v1/delete?query={foo="bar"}&start=1591616227&end=1591619692' \
+  -H 'x-scope-orgid: 1'
+```
```

Comment:
nit: For other examples we use the `bash` code block.

```suggestion
```bash
$ curl -g -X POST \
    'http://127.0.0.1:3100/loki/api/v1/delete?query={foo="bar"}&start=1591616227&end=1591619692' \
    -H 'x-scope-orgid: 1'
```
```

I actually don't really know when we use it, and when not.